### ISSUE TITLE
error :overflow encountered in exp and multiply

### ISSUE THREAD

**Issue by x-Sapphire** [chenyuntc/simple-faster-rcnn-pytorch#171]
/simple-faster-rcnn-pytorch-master/model/utils/bbox_tools.py:68: RuntimeWarning: overflow encountered in exp
  h = xp.exp(dh) * src_height[:, xp.newaxis]
/simple-faster-rcnn-pytorch-master/model/utils/bbox_tools.py:68: RuntimeWarning: overflow encountered in multiply
  h = xp.exp(dh) * src_height[:, xp.newaxis]
/simple-faster-rcnn-pytorch-master/model/utils/bbox_tools.py:69: RuntimeWarning: overflow encountered in exp
  w = xp.exp(dw) * src_width[:, xp.newaxis]
/simple-faster-rcnn-pytorch-master/model/utils/bbox_tools.py:69: RuntimeWarning: overflow encountered in multiply
  w = xp.exp(dw) * src_width[:, xp.newaxis]

and in the visdom I cannot see any loss with cls or loc

**Comment by liruixl** [comment]
hello，how do you solve it? I met it too

**Comment by shida666** [comment]
got the same problem, do you solve it?? thank you!

**Comment by shida666** [comment]
> hello，how do you solve it? I met it too

got the same problem, do you solve it?? thank you!

**Comment by PeterBishop0** [comment]
same!Help!

**Comment by jaypmorgan** [comment]
I was able to mitigate this issue by using the Adam optimizer (setting the `use_adam` bool in config to `True`).

I hope this helps.

**Comment by ZichengDuan** [comment]
> I was able to mitigate this issue by using the Adam optimizer (setting the `use_adam` bool in config to `True`).
> 
> I hope this helps.

It works! Thanks.
This is a rather wired problem, I couldn't figure out why the bbox_tools is affected by optimizers.

**Comment by ZichengDuan** [comment]
Updates.
I changed the optimizer to Adam according to jaypmorgan's suggestion. The training and loss curves seem normal now, however, I am still encountering these warnings, even though my training process is not influenced by these warnings...

**Comment by aiyodiulehuner** [comment]
got the same problem
